src/App.js

```diff
@@ -1,25 +1,79 @@
-import logo from './logo.svg';
-import './App.css';
+// Modules
+import { useEffect, useState } from "react";
+import axios from "axios";
+// Styling
+import "./App.css";
+// Components
+import { FetchedDataDisplay } from "./components/FetchedDataDisplay";
+
+const App = () => {
+  const date = () => {
+    let months = [
+      "January",
+      "February",
+      "March",
+      "April",
+      "May",
+      "June",
+      "July",
+      "August",
+      "September",
+      "October",
+      "November",
+      "December",
+    ];
+    let days = [
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Satuday",
+      "Sunday",
+    ];
+  };
+
+  const [cityName, setCityName] = useState("");
+  const [shouldFetch, setShouldFetch] = useState(false);
 
-function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>Welcome to the weather app</h1>
+      <h2>Get the right temperature and dress according to the weather:</h2>
+
+      {/* Search bar to grab user location */}
+      <form
+        action=""
+        onSubmit={(e) => {
+          e.preventDefault();
+          if (cityName) {
+            setShouldFetch(true);
+          }
+        }}
+      >
+        <input
+          type="text"
+          className="search-bar"
+          placeholder="Select location..."
+          value={cityName}
```

Comment:
Might want to add a state variable to hold the user's input separate from the cityName state, which could get its value transferred to cityName after submit.